contracts/deployables/freeze-voting/FreezeVotingBase.sol

```diff
@@ -244,7 +244,8 @@ abstract contract FreezeVotingBase is
         FreezeVotingBaseStorage storage $ = _getFreezeVotingBaseStorage();
 
         // Start new freeze proposal
-        $.freezeProposalCreated = uint48(block.timestamp); // Mark creation time
+        // Use previous timestamp to ensure ERC5805 getPastVotes works
+        $.freezeProposalCreated = uint48(block.timestamp - 1); // Mark creation time
```

Comment:
This was a bug introduced a few PRs ago, it should have been `block.timestamp - 1` the whole time